src/Components/Toast/toast.js

```diff
@@ -0,0 +1,23 @@
+import { useEffect } from "react";
+import { useDataContext } from "../../Context/data-context";
+
+export const Toast = () => {
+  const {state:{toastMsg}, dispatch} = useDataContext();
+
+  const  closeToast = () => {
+    console.log(toastMsg); //undefined -- FIX IT
```

Comment:
you can remove console here

Reply:
Thanks for this observation. I had missed it completely